metricbeat/docs/modules/googlecloud.asciidoc

```diff
@@ -0,0 +1,72 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-googlecloud]]
+[role="xpack"]
+== Google Cloud Platform module
+
+beta[]
+
+This module periodically fetches monitoring metrics from AWS Stackdriver using
+https://cloud.google.com/monitoring/api/metrics_gcp[Stackdriver Monitoring API] for Google Cloud Platform services.
+Note: extra GCP charges on Stackdriver Monitoring API requests will be generated by this module.
+
+
+[float]
+=== Example configuration
+
+The Google Cloud Platform module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: googlecloud
+  metricsets:
+    - compute
+    - pubsub
+    - firestore
+    - firebase
+    - storage
+    - loadbalancing
+  zone: "your zone"
```

Comment:
can we specify more than one zone here?

Reply:
Good question. We can't specify it right now. The idea was to maintain first version as simple as possible. It's actually possible to request all metrics for a project without zone filter or even request various zones but we are moving slow yet and see how it goes because the code to request metrics and convert them using lightweight modules is pretty complex already.

Reply:
perhaps it would be interesting to put a real zone here so things won't fail if they start the module out of the box?

Reply:
I prefer that things do fail explicitly so that a user with machines in Europe that runs Metricbeat will have an specific error saying `zone "your zone" not found` instead of silent errors of simply not sending any event because there are no machines in that zone/region which may lead to think that Metricbeat is not working properly (it's your fault because you didn't set the correct zone, but that's implicit)